pyface/tasks/action/api.py

```diff
@@ -58,6 +40,30 @@
 - :class:`~.TaskWindowToggleGroup`
 - :class:`~.CreateTaskWindowAction`
 
+ActionSchemas and aliases
+-------------------------
+
+Import of these objects from pyface.tasks.actions.api is deprecated and will
+be removed in a future Pyface release.  Instead use pyface.action.schema.api.
```

Comment:
Maybe we want to raise the deprecation warnings on this PR?  Or it may be too soon and comments are fine for now - but if we don't raise the warnings it would be good to open some GH issues so these are less likely to be forgotten

Reply:
My thought was it's too soon - we don't want to issue warnings until we've had a chance to fix upstream packages otherwise people will get warnings that they can't do anything about.

Reply:
Makes sense! I've opened #1077